benchmark/v8/get-stats.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common.js');
+const v8 = require('v8');
+
+const bench = common.createBenchmark(main, {
+  method: [
+    'getHeapStatistics',
+    'getHeapSpaceStatistics'
+  ],
+  n: [1e6],
+  flags: ['--ignition --turbo', '']
```

Comment:
I'll just back the flags bit out of this particular change to keep it from holding this up at all, but big +1 on refactoring this generally.